app/login/page.tsx

```diff
@@ -52,31 +85,65 @@ const Login = (): JSX.Element => {
             with a league
           </p>
         </div>
-        <div id="input-container" className="grid gap-4">
-          <Input
-            data-testid="email"
-            type="email"
-            value={email}
-            placeholder="Email"
-            onChange={handleEmail}
-          />
-          <Input
-            data-testid="password"
-            type="password"
-            value={password}
-            placeholder="Password"
-            onChange={handlePassword}
-          />
-          <Button
-            data-testid="continue-button"
-            label="Continue"
-            disabled={!email && !password}
-            onClick={() => loginAccount({ email, password })}
-          />
-          <LinkCustom href="/register">
-            Sign up to get started with a league
-          </LinkCustom>
-        </div>
+        <Form {...form}>
+          <form
+            id="input-container"
+            className="grid gap-4"
```

Comment:
This should be `gap-3` according to the spacing in the design.

Reply:
Updated to gap - 3